src/main/java/tars/Storage.java

```diff
@@ -0,0 +1,116 @@
+package tars;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Handles the storing of the Tasks added to the list and retrieves information of Tasks when reopened
+ *
+ * @author csk
+ * @version 1
+ */
+public class Storage {
+    protected static File taskFile;
+
+    /**
+     * Constructor for Storage object which takes in a filePath and sets filePath field in object
+     *
+     * @param filePath
+     * @throws IOException
+     */
+    public Storage(String filePath) throws IOException {
+        this.taskFile = new File(filePath); // create a File for the given file path
+
+        if (!taskFile.exists()) {
+            taskFile.getParentFile().mkdirs();
+            taskFile.createNewFile();
+        }
+    }
+
+    /**
+     * Writes file that will be used for the Tars application based on tasks existing or changed in tasksList
```

Comment:
you could be more concise with this header comment, e.g. something like
Writes existing tasks or changes in the tasklist to file